jenkins-node/docker-images/static-analysis/Dockerfile

```diff
@@ -1,38 +1,16 @@
-#######
-# cppcheck build
-######
-
-FROM alpine:latest AS cppcheck_build
-
-# Version 1.90
-ARG cppcheck_commit=077e652
-
-RUN apk add git ninja cmake g++ pcre-dev && \
-    git clone https://github.com/danmar/cppcheck.git
-
-WORKDIR /cppcheck
-RUN git checkout ${cppcheck_commit} && mkdir build
-
-WORKDIR /cppcheck/build
-RUN CXXFLAGS="--static" cmake .. -DCMAKE_BUILD_TYPE=Release -DHAVE_RULES=ON \
-    -DUSE_MATCHCOMPILER=ON -G Ninja && \
-    cmake --build .
-
 FROM jenkins/jnlp-slave
 
 ARG FLAKE_VERSION=3.7.9
 ARG PEP_VERSION=1.7.1
 
-COPY --from=cppcheck_build /cppcheck/build/bin/cppcheck /usr/bin/
-COPY --from=cppcheck_build /cppcheck/cfg /usr/local/share/Cppcheck/cfg
-
 USER root
 
 RUN apt-get update
 RUN apt-get install -t stretch-backports cmake -y --no-install-recommends
 
 # Ordered in probability of each updating from least->most
 RUN apt-get install clang-format-6.0 parallel -y --no-install-recommends
+RUN apt-get install cppcheck -y --no-install-recommends
```

Comment:
I think `jenkins/jnlp-slave` must be based on an older version of Debian as
```shell
cppcheck --version
```
gives `Cppcheck 1.76.1` but we currently have 1.82. Can we put back the build code you have and just pick an appropriate commit like [this](https://github.com/danmar/cppcheck/commit/23b253e9eb3296a03afd5d52908726da1231f0dd)?